index.html

```diff
@@ -59,5 +57,159 @@ <h1>
+        <button class="button2">See Project</button>
+      </div>
+      <div class="card2">
+        <img src="resources/Img Placeholder (1).png" alt="" />
+        <div class="card-text">
+          <h2>Profesional Art Printing Data</h2>
+          <p>
+            A daily selection of privately personalized reads; no accounts or
+            sign-ups required. Has been the industry's standard.
+          </p>
+          <div class="box2">
+            <p>HTML</p>
+            <p>Bootstrap</p>
+            <p>Ruby</p>
+          </div>
+        </div>
+        <button class="button2">See Project</button>
+      </div>
+      <div class="card2">
+        <img src="resources/Img Placeholder (1).png" alt="" />
+        <div class="card-text">
+          <h2>Profesional Art Printing Data</h2>
+          <p>
+            A daily selection of privately personalized reads; no accounts or
+            sign-ups required. Has been the industry's standard.
+          </p>
+          <div class="box2">
+            <p>HTML</p>
+            <p>Bootstrap</p>
+            <p>Ruby</p>
+          </div>
+        </div>
+        <button class="button2">See Project</button>
+      </div>
+      <div class="card2">
+        <img src="resources/Img Placeholder (1).png" alt="" />
+        <div class="card-text">
+          <h2>Profesional Art Printing Data</h2>
+          <p>
+            A daily selection of privately personalized reads; no accounts or
+            sign-ups required. Has been the industry's standard.
+          </p>
+          <div class="box2">
+            <p>HTML</p>
+            <p>Bootstrap</p>
+            <p>Ruby</p>
+          </div>
+        </div>
+        <button class="button2">See Project</button>
+      </div>
+      <div class="card2">
+        <img src="resources/Img Placeholder (1).png" alt="" />
+        <div class="card-text">
+          <h2>Profesional Art Printing Data</h2>
+          <p>
+            A daily selection of privately personalized reads; no accounts or
+            sign-ups required. Has been the industry's standard.
+          </p>
+          <div class="box2">
+            <p>HTML</p>
+            <p>Bootstrap</p>
+            <p>Ruby</p>
+          </div>
+        </div>
+        <button class="button2">See Project</button>
+      </div>
+    </section>
+    <section class="about">
+      <h2>About me</h2>
+      <p>
+        Hello I’m a software developer! I can help you build a product , feature
+        or website Look through some of my work and experience! If you like what
+        you see and have a project you need coded, don’t hesitate to contact me.
+      </p>
+      <button class="button1">Get my resume</button>
+      <div class="underline"></div>
+    </section>
+    <section class="language">
+      <div class="lang">
+        <img src="resources/Rectangle 55.png" alt="" />
+        <h3>Language</h3>
+      </div>
+      <p>Javascript</p>
+      <p>Ruby</p>
+      <p>HTML</p>
+      <p>CSS</p>
```

Comment:
- [ ] The technology categories ("Languages", "Frameworks" and "Skills") are in a ul, each of them in a li
- [ ] The technology in each category(e.g., "JavasScript", "HTML" and "CSS") are also a list ul and each of them a li